Source/Chronozoom.UI/scripts/authoring.js

```diff
@@ -140,6 +140,49 @@ var CZ;
         }
 
         /**
+        * The main function to test a timeline on intersections if it has manually selected height.
+        * It calculate the neccessary height which must be added to place timeline.
+        * @param  {Object} tp       An estimated parent timeline.
+        * @param  {Object} tc       An estimated child timeline. This one will be tested.
+        * @param  {Boolean} editmode If true, it doesn't take into account edited timeline.
+        * @return {Number}          A necessary value.
+        */
+        function necessarySupplement(tp, tc, editmode) {
+            var usedY = [];
+
+            if (editmode && Authoring.selectedTimeline.parent.children) {
+                Authoring.selectedTimeline.parent.children.forEach(function (ael) {
+                    if (ael.offsetY === null) {
+                        if (!(tc.x >= ael.x + ael.width
+                            || ael.x >= tc.x + tc.width)) {
+                            if ((ael.y !== tc.y) && (ael.height !== tc.height))
+                                usedY.push({ top: ael.y + ael.height, bottom: ael.y });
+                        }
+                    }
+                });
+            }
+
+            var locMin = tc.y;
+            var locMax = Number.MIN_VALUE
+
+            //Finding the area of intersection
+            usedY.forEach(function (ael) {
+                if (!((ael.top <= (tc.y))
+                    || (tc.y + tc.height <= ael.bottom))) {
+                    arranged = false;
+                    if (ael.top > locMax)
+                        locMax = ael.top;
+                }
+            });
+            
+            if (locMax > locMin)
+                return (locMax - locMin) * (1.001 / (Math.max(tc.offsetY,
```

Comment:
This is inverse proportion. I added epsilon just in case there are problems. But I think it might be replaced by 1.0